gamedrake/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Slim Moe",
+      "id": "686244278304899077"
+    },
+    "service": "Gamedrake",
+    "description": {
+        "en": "Game Drake is the platform for players to learn about games.",
+        "tr": "Gamedrake, oyunlar hakkında bilgi edinebileceğiniz oyuncu platformu." 
+       },
+    "url": ["gamedrake.com"],
```

Comment:
Please make this a string, it does not need to be an array as it only contains one thing.

`["gamedrake.com"]` -> `"gamedrake.com"`